examples/tools/microsoft/msbuild/create_your_first_package.rst

```diff
@@ -0,0 +1,168 @@
+  file, that the project must add to his properties. If the project had dependencies with Conan
+  ``requires``, it should add ``MSBuildDeps`` too and add the relevant generated files property
+  sheets.
+- The ``build()`` method uses the ``MSBuild()`` helper to drive the build of the solution
+- As the project doesn't have any "install" functionality in the build scripts, the ``package()``
+  method can manually define which files must be copied.
+
+The ``hello.vcxproj`` project file adds the generated property sheets like ``conantoolchain.props``
+to the project, so the build can receive the Conan input ``settings`` and act accordingly.
+
+.. code-block:: xml
+   :caption: hello.vcxproj
+
+    <ImportGroup Label="PropertySheets">
+      <Import Project="conan\conantoolchain.props" />
+    </ImportGroup>
+
+If the project had dependencies, it should add the dependencies generated ``.props`` files too.
+
+
+The **test_package** folder also contains a ``test_hello.vcxproj`` file, that includes both the
+toolchain and the dependencies property sheets:
+
+.. code-block:: xml
+  :caption: test_package/test_hello.vcxproj
+
+  <ImportGroup Label="PropertySheets">
+      <Import Project="conan\conantoolchain.props" />
+      <Import Project="conan\conandeps.props" />
+  </ImportGroup>
+
+
+Note the ``test_package/conanfile.py`` contains also a ``generators="MSBuildDeps"``.
+
+Let's build the package from sources with the current default configuration, and then let
+the ``test_package`` folder test the package:
+
+.. code-block:: bash
+
+    $ conan create .
+    
+    ...
+    ======== Testing the package: Executing test ========
+    hello/1.0 (test package): Running test()
+    hello/1.0 (test package): RUN: x64\Release\test_hello
+    hello/1.0: Hello World Release!
+      hello/1.0: _M_X64 defined
+      hello/1.0: MSVC runtime: MultiThreadedDLL
+      hello/1.0: _MSC_VER1939
+      hello/1.0: _MSVC_LANG201402
+      hello/1.0: __cplusplus199711
+    hello/1.0 test_package
+
+
+We can now validate that the recipe and the package binary are in the cache:
+
+
+.. code-block:: bash
+
+    $ conan list hello/1.0:*
+    Local Cache:
+      hello
+        hello/1.0
+          revisions
+            856c535669f78da11502a119b7d8a6c9 (2024-03-04 17:52:39 UTC)
+              packages
+                c13a22a41ecd72caf9e556f68b406569547e0861
+                  info
+                    settings
+                      arch: x86_64                  
+                      build_type: Release           
+                      compiler: msvc                
+                      compiler.cppstd: 14           
+                      compiler.runtime: dynamic     
+                      compiler.runtime_type: Release
+                      compiler.version: 193         
+                      os: Windows                   
+         
+               
+
+
+
+
+.. seealso::
```

Comment:
I think we should change all ReadMore for ``seealso``:

- The ReadMore section sometimes slips in Table of Contents, and it results very ugly
- It doesn't allow to introduce more inline links, only at the very end of the section, while sometime some more inline links at the end of a subsection would be useful